src/Packetery/Module/Checkout.php

```diff
@@ -577,11 +578,12 @@ function () {
 	public function updateShippingRates(): void {
 		$customRates = $this->getShippingRates();
 
-		$packages = WC()->shipping()->get_packages();
+		$isCurcyActive = $this->isCurcyPluginActive();
+		$packages      = WC()->shipping()->get_packages();
 		foreach ( $packages as $i => $package ) {
 			if ( ! empty( $package['rates'] ) ) {
 				foreach ( $package['rates'] as $key => $rate ) {
-					if ( isset( $customRates[ $rate->get_id() ] ) ) {
+					if ( isset( $customRates[ $rate->get_id() ] ) && ! $isCurcyActive ) {
```

Comment:
Jsme si jistí, že Curcy a Woocs nejdou používat zároveň?

Reply:
teoreticky to mozne je, prakticky ne, protoze se ti tim rozdrbe prepocitavani uplne vseho

Reply:
Ok, kolizi tedy nepředpokládáme.